test/helper.rb

```diff
@@ -7,10 +7,18 @@
 I18n.load_path << File.expand_path("../locale/en.yml", __FILE__)
 
 class User < Minidoc
+  self.collection_name = "users"
```

Comment:
It's not something we'd do in normal code, so I don't think it's necessary here in the test either. The child class is right below it, after all. NBD if you want to keep it, though.